group_vars/all.yml

```diff
@@ -823,4 +826,14 @@ vd_users:
   - username: "{{ ansible_nas_user }}"
     password: "topsecret"
     sudo: "Y"
-vd_rdp_port: 3389
\ No newline at end of file
+vd_rdp_port: 3389
+
+###
+### Gotify
+###
+gotify_available_externally: "false"
+gotify_data_directory: "{{ docker_home }}/gotify/config"
+gotify_port_http: "8077"
+gotify_port_https: "8078"
```

Comment:
This port doesn't appear to be mapped to the container?

Reply:
Fixed - yeah this was pushed when I didn't have Traefik setup yet. :)